res/schema.xml

```diff
@@ -489,4 +489,30 @@ METADATA
       ALTER TABLE library ADD COLUMN color INTEGER;
     </sql>
   </revision>
+  <revision version="32" min_compatible="3">
+    <description>
+      Convert the PredefinedColor ID to the actual RGB value.
+    </description>
+    <sql>
+      <!-- No Color becomes Orange (not in default palette) -->
```

Comment:
At last not in the 2.3 alpha palette. 

Do we need to add it to the default palette? 
Is returning to default color from a previous set color a use case.

Reply:
> At last not in the 2.3 alpha palette.
> 
> Do we need to add it to the default palette?
> Is returning to default color from a previous set color a use case.

If someone want to do that he can after #2530 has been merged.